### ISSUE TITLE
Recruiting an Editorial Team

### ISSUE THREAD

**Issue by janl** [hoodiehq/discussion#82]
Heya everyone!

I’d like to propose that we recruit an editorial team for the Hoodie community.

Primarily, I’d like a group of people to be responsible for our project communication. That’s mainly our blog (http://hood.ie/blog), but we also have a tumblr, a twitter, maybe we find someone to run a Facebook page for us, etc.

In the past, we had individuals contributing things like the weekly TGIF, but the setup was not really sustainable, so from now on, I’d like to have a team work on this together, rather than an individual. (All these people have done is HIGHLY appreciated, of course <3 :).

I see these areas of activity for the editorial team:

1. find more editorial team members (the more people who work on this, the less everyone has to do, and collectively, we get more done :)
2. plan and write blog posts on or around Hoodie, Hoodie releases, side projects, TGIF, community member profiles, conference and meetup recaps, etc.
3. solicit and curate contributions to our blog from the larger community.
4. plan and manage our Twitter account, announce new content, new people, wish happy birthday, help funnel support requests to the engineering team, etc.
5. coordinate with @qate_oh on doing PR around The Hoodie Letter and The Offline First Reader releases.
6. whatever else you think is fun!

* * *

For the first round of this issue, I’d like to discuss with you whether you think this is a good idea, and if not, what could be improved to achieve a similar goal, or whatever else we can do in this context.

In a next step, I’d like to take the result from this issue with all the feedback in it and open another issue that we can actively use tor recruit people and hand out on social media and to our friends. On that note, how can we write a “job ad” that sounds appealing to people who are good at this? :)

What do you think?

- [ ] solicit feedback
- [ ] write recruiting issue

**Comment by Charlotteis** [comment]
I agree!

Also, I believe at some point I offered to help write some blog posts/help others write blog posts/proof blog posts for Hoodie but not sure if that went anywhere. The offer still stands!

I think that we have the 'product' but we aren't so good at telling people about it. Having people dedicated to doing so would be great. I'd love to have interviews with the community, write-ups about new features, challenges we face at Hoodie and a much more active Twitter account.

The job advert itself can be it's own Pull Request, where we all throw in our ideas and make edits. At my work I made a new job advert and then went round to all my team members and asked them to read/make edits and went back and forth until it was done. Things like alexjs and joblint can help with this to get the language friendly.

**Comment by janl** [comment]
> The job advert itself can be it's own Pull Request,

Excellent idea! :)

**Comment by janl** [comment]
> Also, I believe at some point I offered to help write some blog posts/help others write blog posts/proof blog posts for Hoodie but not sure if that went anywhere. The offer still stands!

@Charlotteis that is one of the reasons I’d like to get this going. In theory, there should be a group of people here that would feel empowered to take that offer and help getting your posts published. As it stands, you’d have to figure it out yourself (after all, you *could* just send a PR to the hood.ie repo, but that’s not very friendly or accessible, or reassuring wrt to a posting timeline or something) and I’d like to change that :)

**Comment by patriciagarcia** [comment]
I think this is a great idea :). I agree with @Charlotteis in the "we haven the 'product' but we aren't so good at telling people about it" and my feeling is that doing this properly requires dedicated people.

**Comment by Charlotteis** [comment]
It would be great if we could write a blog post without needing to do a PR too, using a more accessible web interface, but I suspect that it is a lot of work...

**Comment by janl** [comment]
> It would be great if we could write a blog post without needing to do a PR too, using a more accessible web interface, but I suspect that it is a lot of work...

I’d shy away from too much infrastructure work at *this* stage, but this is a great point. We link to http://issuetemplate.com/#/hoodiehq/bugs/issues from http://hood.ie/bug/ — a similar thing could be set up to make writing posts more easy. I think the “editorial team”, once it exists, should also be responsible for the tools it uses to produce their work (or at least demand better tools from the larger community, if they feel something blocks them).

**Comment by Charlotteis** [comment]
Did not know either of those links existed, thanks! And yes I agree, we should not wait for this to be built before we start producing content. But it'd be a great thing to do, maybe a nice single page ember app or something like that /shrug.

**Comment by janl** [comment]
Alright, thanks all for your input! I’ve started a draft for the job advert and I need some help with it, especially the “getting started” bit at the end. Any input is welcome! — https://docs.google.com/document/d/1dB_1GRn5RuBhXvqOuYD4uvHWPUNlKRZSAoWRFWOiWrA/edit

**Comment by gr2m** [comment]
can this be closed now? In favor of https://github.com/hoodiehq/discussion/issues/84
